src/task.py

```diff
@@ -62,16 +42,11 @@ class SCLPolygons(SCLTask):
         "reduce_res_input_pixels": 0.5,
         "structural_habitat_patch_size": 5,  # sq km
         "hii": {
-            "zone_1": 10,
-            "zone_2": 10,
-            "zone_3": 10,
-        },  # TODO: use this as a default value, if dynamic thresholding fails
-        "probability": 1,  # TODO: probability threshold may change based on final probability format
-        "connectivity_distance": 2,  # km (1/2 of actual dispersal distance)
-        "landscape_size": 3,
-        "current_range": 2,
-        "landscape_probability": 1,
-        "landscape_survey_effort": 1,
+            "zone_1": 18,
+            "zone_2": 8,
+            "zone_3": 5,
+        },  # TODO: possibly replace with dynamic thresholding
+        "connectivity_distance": 4,
```

Comment:
How about `dispersal_distance` and then right before 207 doing something like
```
connectivity_distance = self.thresholds["dispersal_distance"] / 2
```
and using that thereafter?